ui/src/dashboards/containers/DashboardPage.tsx

```diff
@@ -450,38 +445,31 @@ class DashboardPage extends Component<Props, State> {
   ): ((value: TempVarsModels.TemplateValue) => void) => (
     value: TempVarsModels.TemplateValue
   ): void => {
-    const {dashboard, location} = this.props
-
-    const currentTempVar = dashboard.templates.find(
-      tempVar => tempVar.id === templateID
-    )
-    const strippedTempVar = stripTempVar(currentTempVar.tempVar)
+    const {
+      dashboard,
+      source,
+      templateVariableLocalSelected,
+      updateTemplateQueryParams,
+      rehydrateNestedTemplatesAsync,
+    } = this.props
 
-    const updatedQueryParam = {
-      [strippedTempVar]: value.value,
-    }
-    this.props.syncURLQueryParamsFromQueryParamsObject(
-      location,
-      updatedQueryParam
-    )
-    this.props.templateVariableLocalSelected(dashboard.id, templateID, [value])
+    templateVariableLocalSelected(dashboard.id, templateID, [value])
```

Comment:
This function is never invoked with more than one value, could you change it to accept just `value` instead of `[value]`. I think it catches a problem in one of the tests too.. If you don't feel like it I can do it in a separate PR, let me know :)

Reply:
will do :)